qemu/tests/qemu_guest_agent.py

```diff
@@ -243,6 +246,34 @@ def gagent_check_install(self, test, params, env):
         pass
 
     @error_context.context_aware
+    def gagent_check_install_uninstall(self, test, params, env):
+        """
+        Repeate install/uninstall qemu-ga package in windows guest
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environmen.
```

Comment:
environmen -> environment

Reply:
fix the typo.
could you check it again,thanks.